lib/hobo_fields/field_declaration_dsl.rb

```diff
@@ -3,8 +3,14 @@
 require 'active_support/proxy_object'
 
 module HoboFields
+  class FieldDeclarationDsl < BasicObject # avoid Object because that gets extended by lots of gems
+    include ::Kernel                      # but we need the basic class methods
 
-  class FieldDeclarationDsl < ActiveSupport::ProxyObject
```

Comment:
Turns out `ProxyObject` is so stripped down, it doesn't even support class methods like `lambda` and `const_get` (used for all constant resolution unless the constant starts with `::`)! I couldn't use `Object` because other gems mix into it (`DelayedJob` messed us up by adding a method that collided with a field name!). But starting with `BasicObject` and including `Kernel` got me a stable foundation to `undef` instance methods from. :-\